win32/Lib/win32timezone.py

```diff
@@ -673,25 +711,22 @@ def _inDaylightSavings(self, dt):
 
         return in_dst
 
-    def GetDSTStartTime(self, year):
+    def GetDSTStartTime(self, year: int) -> datetime.datetime:
         "Given a year, determines the time when daylight savings time starts"
         return self.getWinInfo(year).locate_daylight_start(year)
 
-    def GetDSTEndTime(self, year):
+    def GetDSTEndTime(self, year: int) -> datetime.datetime:
         "Given a year, determines the time when daylight savings ends."
         return self.getWinInfo(year).locate_standard_start(year)
 
-    def __le__(self, other):
-        return self.__dict__ < other.__dict__
```

Comment:
In https://github.com/mhammond/pywin32/pull/1990 I updated `__cmp__` for `__le__`, but that doesn't make sense for dicts. Comparisons of `TimeZoneInfo` has been broken since Python 3 anyway so removing this doesn't remove any functionality.